services/workflows-service/package.json

```diff
@@ -10,6 +10,7 @@
     "lint": "eslint . --fix",
     "start": "nest start",
     "dev": "npm run start:watch",
+    "prod": "npm run db:migrate-up && node dist/main",
```

Comment:
Do the examples work after this change? (try this one which is relevant for dev https://github.com/ballerine-io/ballerine/blob/main/websites/docs/src/pages/en/learn/back-office.md)

Reply:
It doesn't affect the example at all. It's a new command that is used only when we initiate the container, which happens only on deployment.